install-shipwright.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+#
+# Installs Shipwright Build Controller and Build-Strategies using the Makefile target.
+#
+
+set -x
```

Comment:
I guess you added this for debugging only?

Reply:
Yes! Let me remove it, please.